lib/json5-serialize.js

```diff
@@ -37,13 +15,14 @@ serialize.types = {
   array(array) {
     let result = '[';
 
-    for (let index = 0; index < array.length; index++) {
+    const len = array.length;
```

Comment:
In this place we cached 'len' twice, see line 25 and `serialize` is a place for performance optimization.